Review in figma (#4686)
sourcegraph/about: handbook/product/design/reviewing_design/index.md

As a teammate outside the design team, it’s your responsibility to understand [when and how your feedback will be most valuable](../../../communication/seeking-and-giving-feedback.md#giving-feedback).

We have a [set of guidelines for how to seek and give great feedback](../../../communication/seeking-and-giving-feedback.md).

## Reviews in Figma

To make reviewing designs easier, we use the following tools in Figma:

> - [**Labels:**](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/🛠Project-Tools?node-id=1502%3A6598) to indicate the state of the **design**: draft, ready for review, current (approved, ready for dev), deprecated (usually left there for historical context), conceptual (exploratory, an alternative solution) or **copy**: draft copy, current copy (approved, the most recent version).

Reviewer: ```suggestion
- [**Labels:**](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/🛠Project-Tools?node-id=1502%3A6598) to indicate the state of specific design elements or views **design**: draft, ready for review, current (approved, ready for dev), deprecated (usually left there for historical context), conceptual (exploratory, an alternative solution) or **copy**: draft copy, current copy (approved, the most recent version). Note that these labels are to provide more granular context than the status of the design project/file as a whole.
```

Addressing @jjinnii's feedback about clarifying these labels.

> - [**Labels:**](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/🛠Project-Tools?node-id=1502%3A6598) to indicate the state of the **mockups**: draft, ready for review, current (approved, ready for dev), deprecated (usually left there for historical context), conceptual (exploratory, an alternative solution) or **copy**: draft copy, current copy (approved, the most recent version).

Reviewer 2: ```suggestion
- [**Labels:**](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/🛠Project-Tools?node-id=1502%3A6598) to indicate the state of the **design**: draft, ready for review, current (approved, ready for dev), deprecated (usually left there for historical context), conceptual (exploratory, an alternative solution) or **copy**: draft copy, current copy (approved, the most recent version).
```

Reviewer 2: @AlicjaSuska what does this mean?

> or **copy**: draft copy, current copy (approved, the most recent version)

Are you referring to copy of a Figma file or copywriting? I'm unaware of how these would be or are used in our current workflows.

Author: I think @quinnkeast added those labels. In my understanding, in copy-heavy designs, we should distinguish between final and in-progress/example copy, if needed. @quinnkeast could you please confirm?

- [**Helper cards:**](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/🛠Project-Tools?node-id=2814%3A6218) to annotate the design and highlight important details. Types of cards: annotation (to provide additional context), todo, feedback requested (to give more context regardign what feedback designer is looking for) and metric collection (see [defining metrics](https://about.sourcegraph.com/handbook/product/design/metrics/defining-metrics)).

> - [**Figma comments:**](https://help.figma.com/hc/en-us/articles/360041068574-Add-comments-to-files) for ongoing discussions with reviewers. Please note: to review longer copy, use Google Doc instead of Figma comments.

Reviewer 3: Could we provide more clarification on what longer copy might mean here? I'm not sure myself :)

> - [**Figma comments:**](https://help.figma.com/hc/en-us/articles/360041068574-Add-comments-to-files) for ongoing discussions with reviewers. Please note: to review longer copy, use Google Doc instead of Figma comments.

Reviewer: ```suggestion
- [**Figma comments:**](https://help.figma.com/hc/en-us/articles/360041068574-Add-comments-to-files) for ongoing discussions with reviewers. You may find it easier to review and collaborate on copy longer than 1-2 lines (such as a complete modal) using a Google Doc instead of Figma comments.
```

Suggestion based on @jjinnii's feedback. I also suggest changing it to "You may find it easier," because it's not always the case, but rather more about making sure we know it's an option that can help.
